products/consul.md

```diff
@@ -69,6 +69,17 @@ releases:
     latestReleaseDate: 2021-12-15
     releaseDate: 2020-06-18
 
+-   releaseCycle: "1.7"
+    eol: true
+    latest: "1.7.14"
+    latestReleaseDate: 2021-04-05
+    releaseDate: 2020-06-10
```

Comment:
Dates don't match https://github.com/endoflife-date/release-data/blob/1ec69e4ac988054df8a10af579259a3036eb7561/releases/consul.json#L4

Reply:
Based on the following elements: 

- [`v1.7.4`](https://github.com/hashicorp/consul/releases/tag/v1.7.4)
- [`v1.7.14`](https://github.com/hashicorp/consul/releases/tag/v1.7.14)

- releaseDate : `2020-06-10`
- latest : `2021-04-15`